Plugins/SafeDIGenerator/SafeDIGenerateDependencyTree.swift

```diff
@@ -101,9 +158,7 @@ extension Target {
         func createBuildCommands(
             context: XcodeProjectPlugin.XcodePluginContext,
```

Comment:
somewhat annoyingly, `XcodeProjectPlugin.XcodePluginContext` has _not_ been updated at all, and all of its APIs are deprecated. Hopefully an oversight that'll be fixed before Xcode 16.0 goes live.